lm_eval/api/task.py

```diff
@@ -233,19 +234,45 @@ def __init__(
         cache_dir: Optional[str] = None,
         download_mode: Optional[str] = None,
         prompt_template: Optional[promptsource.templates.Template] = None,
+        description: Optional[str] = None,
+        example_separator: str = "\n###\n",
```

Comment:
is this also supposed to be `Optional[str]`?

Reply:
Yup! Good catch